PowerEditor/src/winmain.cpp

```diff
@@ -313,10 +328,10 @@ void doException(Notepad_plus_Window & notepad_plus_plus)
 
 
 
-int WINAPI WinMain(HINSTANCE hInstance, HINSTANCE, LPSTR, int)
+int WINAPI wWinMain(HINSTANCE hInstance, HINSTANCE, PWSTR pCmdLine, int)

```

Comment:
Why do we need `wWinMain` ?

Reply:
I need access to a wide char command line parameter. `GetCommandLineW` does fit my use case, because it also includes the program name. See:

```
lpCmdLine

Type: LPSTR

The command line for the application, excluding the program name. To retrieve the entire command line, use the GetCommandLine function.
```